hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioNetworking.java

```diff
@@ -465,8 +487,9 @@ public Context() {
             }
         }
 
-        public void setSelectionKeyWakeupEnabled(boolean selectionKeyWakeupEnabled) {
+        public Context selectionKeyWakeupEnabled(boolean selectionKeyWakeupEnabled) {
```

Comment:
updated so it is suitable for chaining and renamed according to this class' method naming convention